core/templates/dev/head/profile/profile.html

```diff
@@ -39,6 +39,10 @@ <h1 class="oppia-profile-username"><strong>{{PROFILE_USERNAME}}</strong></h1>
               <span ng-if="!userBio">
                 <em>This user has not supplied a bio yet.</em>
               </span>
+            </br>
+              <span ng-if="firstContributionDatetime">Contributing since 
```

Comment:
Sorry, I didn't understand the original comment. So you mean I should put </br> after the <span....> and before "Contributing since" ?